web_services/foxx/brightid/db.js

```diff
@@ -600,15 +614,19 @@ function linkContextId(id, context, contextId, timestamp) {
     }
   }
 
-  if (getUserByContextId(coll, contextId)) {
-    throw 'contextId is duplicate';
-  }
-
   coll.insert({
     user: id,
     contextId,
     timestamp
   });
+
+  // sponsor the user if contextId is temporarily sponsored
+  const tempSponsorship = sponsorshipsColl.firstExample({ contextId });
```

Comment:
add an index to `contextId` in https://github.com/BrightID/BrightID-Node/pull/176/files#diff-de67713be4d96945f172f3511032a9a3c16e438325f342ca60fa0ad2eb7c4cfdR30-R34